### PR TITLE
docs(UI Builder): fix HTTPS configuration

### PR DESCRIPTION
**Pull request by abirembaut**
* fix volumes
* fix nginx configuration retrieval

Relates to [BPM-271](https://bonitasoft.atlassian.net/browse/BPM-271)

[BPM-271]: https://bonitasoft.atlassian.net/browse/BPM-271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/applications/pages/ui-builder/how-to-configure-https.adoc [hunk 1]
```diff
@@ -41,20 +41,22 @@ services:
...
+      - <path_on_docker_host>/https.conf:/etc/nginx/snippets/https.conf
       - <path_on_docker_host>/nginx.crt:/etc/ssl/certs/nginx.crt
       - <path_on_docker_host>/nginx.key:/etc/ssl/private/nginx.key
       - <path_on_docker_host>/dhparam.pem:/etc/ssl/certs/dhparam.pem
 ----
 
 === Update the nginx configuration to configure HTTPS
 
-Start the service so that the `nginx.conf` file is created on the Docker host (`docker compose up bonita-ui-proxy`). Then, update it to include a conf snippet to the server block:
+First, retrieve the nging configuration from the service `bonita-ui-proxy` using the following steps:
```
Comment (marcobonita) [inline modules/applications/pages/ui-builder/how-to-configure-https.adoc]:
```suggestion
First, retrieve the nginx configuration from the `bonita-ui-proxy` service using the following steps:
```

File: modules/applications/pages/ui-builder/how-to-configure-https.adoc [hunk 2]
```diff
@@ -41,20 +41,22 @@ services:
...
+
+* create the `bonita-ui-proxy` container by running `docker compose create bonita-ui-proxy`
+* list the containers `docker container ps -a` and copy the container name for `bonita-ui-proxy` image (for example `docker-bonita-ui-proxy-1`)
+* copy on the host the nginx configuration form the container `docker cp docker-bonita-ui-proxy-1:/etc/nginx/templates/nginx.conf.template <path_on_docker_host>/nginx.conf`
```
Comment (marcobonita) [inline modules/applications/pages/ui-builder/how-to-configure-https.adoc]:
```suggestion
* Create the `bonita-ui-proxy` container by running `docker compose create bonita-ui-proxy`
* List the containers `docker container ps -a` and copy the container name for `bonita-ui-proxy` image (for example `docker-bonita-ui-proxy-1`)
* Copy on the host the nginx configuration form the container `docker cp docker-bonita-ui-proxy-1:/etc/nginx/templates/nginx.conf.template <path_on_docker_host>/nginx.conf`
```

File: modules/applications/pages/ui-builder/how-to-configure-https.adoc [hunk 3]
```diff
@@ -31,7 +31,7 @@ services:
 To configure HTTPS, you need to mount the certificate and key files into the Bonita UI proxy container. You can do this by mounting volumes.
 You will also need other mounts to update the nginx configuration.
 
-Edit the `docker-compose.yml` file and add the following volumes/bind mounts to the bonita-ui-proxy service (replacing <path_on_docker_host> by the local files paths):
+Edit the `docker-compose.yml` file and add the following volumes to the bonita-ui-proxy service (replacing <path_on_docker_host> by the local files paths):
```
Comment (marcobonita) [inline modules/applications/pages/ui-builder/how-to-configure-https.adoc]:
```suggestion
Edit the `docker-compose.yml` file and add the following volumes to the `bonita-ui-proxy` service (replacing <path_on_docker_host> by the local files paths):
```

**Review by marcobonita — CHANGES_REQUESTED** [review]

**Review by marcobonita — APPROVED** [review]

File: modules/applications/pages/ui-builder/how-to-configure-https.adoc [hunk 4]
```diff
@@ -41,20 +41,22 @@ services:
...
     environment: ...
     volumes:
-      - type: bind
-        source: <path_on_docker_host>/nginx.conf
-        target: /etc/nginx/templates/nginx.conf.template
-      - type: bind
-        source: <path_on_docker_host>/https.conf
-        target: /etc/nginx/snippets/https.conf
+      - <path_on_docker_host>/nginx.conf:/etc/nginx/templates/nginx.conf.template
+      - <path_on_docker_host>/https.conf:/etc/nginx/snippets/https.conf
```
Comment (csouchet) [inline modules/applications/pages/ui-builder/how-to-configure-https.adoc:45]: **question:** 
Is it normal that is not exactly the same content than our docker-compose.yml in UIB repository ?

ℹ️ In our repo:
```
      - ../nginx/https.conf:/etc/nginx/snippets/https.conf
      - ../nginx/https-redirect.conf:/etc/nginx/snippets/https-redirect.conf
      - ../nginx/cookies.conf:/etc/nginx/snippets/cookies.conf
```

Comment (abirembaut) [reply modules/applications/pages/ui-builder/how-to-configure-https.adoc:45]: this is once my PR will be merged.
For the doc of 1.1.0, I decided to keep is simple and not ask to create too many files

**Review by csouchet — APPROVED** [review]

**Pull request merged**